src/deployment-guides/tap.md

```diff
@@ -0,0 +1,799 @@
+# Tanzu Application Platform Deployment Guide
+
+## Executive Summary
+
+This deployment guide outlines the deployment steps for VMware Tanzu Application Platform on a Kubernetes workload cluster. In accordance with the [Tanzu Application Platform Reference Design](../reference-designs/tap-architecture-planning.md), four clusters will be created:
+
+* Tanzu Application Platform Build Cluster 
+* Tanzu Application Platform Run Cluster 
+* Tanzu Application Platform View Cluster 
+* Tanzu Application Platform Iterate Cluster 
+
+## Prerequisites
+
+Before deploying VMware Tanzu Application Platform, ensure that the following prerequisites are met:
+
+* A [Tanzu Network](https://network.tanzu.vmware.com/) account is available for downloading Tanzu Application Platform packages.
+* A container image registry with push and write access (such as Harbor, Docker Hub, or Azure Container Registry) is available. This will be used for application images and runtime dependencies.
+* Network access to [VMware's image registry](https://registry.tanzu.vmware.com) is available.
+<!-- /* cSpell:disable */ -->
+* DNS Records for components like Cloud Native Runtimes (knative) i.e. cnrs, Tanzu Learning Center, Tanzu Application Platform GUI etc.
+* A centralized git repository from GitHub, Gitlab or Azure DevOps for the Tanzu Application Platform GUI's software catalogs, along with a token allowing read access.
+* A Kubernetes workload cluster versions 1.20, 1.21, or 1.22 on Azure Kubernetes Service, Amazon Elastic Kubernetes Service, Google Kubernetes Engine, or [minikube](https://minikube.sigs.k8s.io/docs/start/) providers.
```

Comment:
```suggestion
* A Kubernetes workload cluster versions 1.20, 1.21, or 1.22 exists on Azure Kubernetes Service, Amazon Elastic Kubernetes Service, Google Kubernetes Engine, or [minikube](https://minikube.sigs.k8s.io/docs/start/) providers.
```

Reply:
Can we include Tanzu Kubernetes Grid (TKG) workload clusters in this list?  I know it's a selling point now that TAP can run on latest TKG version (v1.5) clusters, so it would be great to include here.
TKG v1.5 can run Kubernetes 1.20, 1.21, and 1.22 workload clusters.